abandoned_checkout.go

```diff
@@ -0,0 +1,91 @@
+package goshopify
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shopspring/decimal"
+)
+
+const abandonedCheckoutsBasePath = "checkouts"
+
+// AbandonedCheckoutService is an interface for interfacing with the abandonedCheckouts endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/online_store/checkout
```

Comment:
This link doesn't work for me.
Maybe something like https://shopify.dev/docs/api/admin-rest/latest/resources/abandoned-checkouts is better?

Reply:
Thank you for your confirmation.
I fixed the link.
86f7696968fde81ceb9e321be8243b06aa18f8ce

Reply:
@oliver006 
Thank you for your confirmation.
I fixed the link.
https://github.com/bold-commerce/go-shopify/commit/86f7696968fde81ceb9e321be8243b06aa18f8ce